sql_metadb/derived_tables/finance_po_inv_transactions.sql

```diff
@@ -0,0 +1,266 @@
+DROP TABLE IF EXISTS finance_po_inv_transactions;
+
+-- Create a derived table that contains information about the fund distribution in po lines and invoice lines
+-- including information on the transaction and budget.
+CREATE TABLE finance_po_inv_transactions AS
+WITH orders_invoice AS (
+    SELECT
+        -- purchase_order
+        purchase_order.id AS po_id,
+        jsonb_extract_path_text(purchase_order.jsonb, 'poNumber') AS po_number,
+        jsonb_extract_path_text(purchase_order.jsonb, 'workflowStatus') AS po_workflowStatus,
```

Comment:
This query contains a lot of camel-case identifiers.  They should be all in lowercase.  I will fix them here, but please check my work and create a new PR if further changes are needed.